src/sql/workbench/contrib/dashboard/browser/dashboardActions.ts

```diff
@@ -100,6 +104,9 @@ export class OEManageConnectionAction extends Action {
 				}
 			}
 		}
+		else if (actionContext === undefined || actionContext === null) {
+			connectionProfile = TaskUtilities.getCurrentGlobalConnection(this._objectExplorerService, this._connectionManagementService, this._editorService) as ConnectionProfile;
+		}
 
 		if (!connectionProfile) {
 			// This should never happen. There should be always a valid connection if the manage action is called for
```

Comment:
Also update this comment - this could much more easily happen now if there's no "global" connection available when the user executes it via keybinding.

Reply:
```ts
// No valid connection (e.g. if this was triggered without an active context to get the connection from) so just return early
```